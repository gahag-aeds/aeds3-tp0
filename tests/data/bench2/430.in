430 215
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 28 7 30 30 7 21 65 84 31 72 64 53 55 94 45 48 77 34 40 49 98 33 15 50 76 38 34 93 98 76 80 78 83 62 8 42 83 25 79 66 49 95 71 5 41 16 5 19 2 97 68 0 83 35 2 59 74 88 4 24 
Add 27 169
Min 37 179
Add 148 290
Max 40 182
Add 268 410
Min 171 313
Add 263 405
Min 83 225
Sub 188 330
Sum 239 381
Sub 192 334
Sum 212 354
Add 26 168
Sum 163 305
Add 19 161
Max 276 418
Add 151 293
Min 111 253
Sub 16 158
Min 40 182
Add 279 421
Sum 29 171
Add 270 412
Max 183 325
Sub 201 343
Max 164 306
Add 94 236
Sum 217 359
Add 2 144
Max 147 289
Sub 179 321
Min 29 171
Sub 183 325
Max 205 347
Sub 68 210
Sum 43 185
Sub 88 230
Max 209 351
Sub 125 267
Max 275 417
Sub 40 182
Sum 30 172
Sub 78 220
Max 65 207
Sub 71 213
Sum 96 238
Sub 219 361
Sum 222 364
Add 206 348
Sum 234 376
Add 140 282
Min 198 340
Add 142 284
Max 2 144
Add 259 401
Sum 55 197
Sub 44 186
Max 66 208
Add 150 292
Max 46 188
Add 91 233
Min 41 183
Sub 74 216
Sum 152 294
Add 245 387
Min 20 162
Sub 194 336
Min 211 353
Sub 107 249
Max 32 174
Sub 77 219
Max 146 288
Add 227 369
Max 155 297
Add 211 353
Min 10 152
Add 250 392
Max 21 163
Sub 110 252
Max 48 190
Add 134 276
Sum 129 271
Add 124 266
Sum 275 417
Sub 9 151
Max 95 237
Sub 208 350
Max 52 194
Sub 39 181
Max 70 212
Add 97 239
Max 8 150
Add 110 252
Max 49 191
Sub 160 302
Min 233 375
Sub 68 210
Sum 66 208
Sub 156 298
Max 175 317
Sub 263 405
Sum 232 374
Sub 33 175
Sum 202 344
Sub 265 407
Min 122 264
Sub 91 233
Min 107 249
Sub 22 164
Sum 79 221
Sub 154 296
Sum 155 297
Add 86 228
Max 156 298
Sub 21 163
Min 94 236
Sub 129 271
Max 228 370
Add 146 288
Sum 47 189
Sub 176 318
Sum 121 263
Add 54 196
Max 285 427
Add 170 312
Max 213 355
Sub 109 251
Sum 237 379
Add 279 421
Sum 264 406
Add 3 145
Sum 254 396
Add 87 229
Sum 202 344
Sub 48 190
Max 65 207
Sub 202 344
Sum 17 159
Add 171 313
Min 223 365
Add 95 237
Min 203 345
Sub 256 398
Max 181 323
Add 237 379
Sum 276 418
Add 153 295
Min 244 386
Add 95 237
Max 11 153
Add 3 145
Max 270 412
Sub 11 153
Min 57 199
Add 146 288
Min 180 322
Add 269 411
Sum 254 396
Add 8 150
Min 139 281
Sub 109 251
Sum 11 153
Add 272 414
Sum 196 338
Sub 91 233
Max 32 174
Add 139 281
Sum 5 147
Sub 48 190
Sum 22 164
Sub 228 370
Sum 21 163
Sub 103 245
Max 75 217
Add 80 222
Min 218 360
Sub 64 206
Max 233 375
Add 174 316
Sum 158 300
Sub 122 264
Min 177 319
Sub 141 283
Sum 132 274
Add 233 375
Max 286 428
Sub 24 166
Sum 93 235
Sub 132 274
Max 198 340
Sub 111 253
Max 240 382
Sub 87 229
Sum 95 237
Add 148 290
Min 177 319
Add 5 147
Max 286 428
Add 181 323
Sum 218 360
Add 186 328
Min 287 429
Add 238 380
Sum 167 309
Add 75 217
Min 31 173
Add 97 239
Max 17 159
Add 263 405
Min 271 413
Add 174 316
Max 97 239
Add 39 181
Sum 193 335
Add 108 250
