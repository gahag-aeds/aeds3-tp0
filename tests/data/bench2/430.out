0
101
1
1
7315
7111
7483
100
2
2
7215
101
103
7443
104
3
99
7392
99
102
7042
99
6745
7033
7019
-2
99
6991
99
100
-3
7376
-4
0
99
98
98
-5
98
97
7174
7929
99
99
98
98
100
1
6518
99
7507
7163
-5
-6
6170
6942
98
-6
100
5939
6686
102
100
7303
7592
7489
7038
95
6047
0
-1
99
7617
0
99
102
-8
-1
7648
-1
6341
7432
97
6389
6031
5961
96
1
101
7468
-4
6953
104
6312
102
102
5949
-7
104
7782
0
7663
-8
101
6
98
8087
